tests/fafnir_tests/mod.rs

```diff
@@ -145,4 +179,13 @@ pub fn main_test(es_wrapper: ElasticSearchWrapper, pg_wrapper: PostgresWrapper)
         .find(|&p| p.key == "amenity")
         .unwrap();
     assert_eq!(amenity_tag.value, "cafe".to_string());
+
+    // Test Address
```

Comment:
I think it's worth it to detail a bit more what you are doing there

Reply:
done.